helm/syntho-ui/templates/dbs/database-data-2-persistentvolumeclaim.yaml

```diff
@@ -12,5 +12,6 @@ spec:
   resources:
     requests:
       storage: 100Mi
+  storageClassName: {{ .Values.db.storageClassName | default "default" }}
```

Comment:
Here we have a default value of "default", but in the `values.yaml` we set it to `standard`. Should this be either the same default, or change the one in `values.yaml`?

Reply:
it was made `default` in values.yaml